python/ray/serve/tests/conftest.py

```diff
@@ -16,10 +15,5 @@ def serve_instance():
 
 
 @pytest.fixture(scope="session")
-def ray_instance():
-    ray_already_initialized = ray.is_initialized()
-    if not ray_already_initialized:
-        ray.init(object_store_memory=int(1e8))
-    yield
-    if not ray_already_initialized:
-        ray.shutdown()
+def ray_instance(serve_instance):
+    pass
```

Comment:
Why this change? Shouldn't we just use serve_instance in this case?

Reply:
Yeah the change was to avoid the segfault warnings, i just replaced all ray_instance with serve_instance